matter/si91x/siwx917/BRD4338A/autogen/sl_component_catalog.h

```diff
@@ -2,18 +2,55 @@
 #define SL_COMPONENT_CATALOG_H
 
 // APIs present in project
+#define SL_CATALOG_MATTER_ACCESS_CONTROL_PRESENT
+#define SL_CATALOG_MATTER_ADMINISTRATOR_COMMISSIONING_PRESENT
+#define SL_CATALOG_MATTER_BASIC_INFORMATION_PRESENT
+#define SL_CATALOG_MATTER_COLOR_CONTROL_PRESENT
+#define SL_CATALOG_MATTER_DESCRIPTOR_PRESENT
+#define SL_CATALOG_MATTER_DIAGNOSTIC_LOGS_PRESENT
+#define SL_CATALOG_MATTER_FIXED_LABEL_PRESENT
+#define SL_CATALOG_MATTER_GENERAL_COMMISSIONING_PRESENT
+#define SL_CATALOG_MATTER_GENERAL_DIAGNOSTICS_PRESENT
+#define SL_CATALOG_MATTER_GROUP_KEY_MGMT_PRESENT
+#define SL_CATALOG_MATTER_GROUPS_PRESENT
+#define SL_CATALOG_MATTER_IDENTIFY_PRESENT
+#define SL_CATALOG_MATTER_LEVEL_CONTROL_PRESENT
+#define SL_CATALOG_MATTER_LOCALIZATION_CONFIGURATION_PRESENT
+#define SL_CATALOG_MATTER_NETWORK_COMMISSIONING_PRESENT
+#define SL_CATALOG_MATTER_ON_OFF_PRESENT
+#define SL_CATALOG_MATTER_OPERATIONAL_CREDENTIALS_PRESENT
+#define SL_CATALOG_MATTER_OTA_REQUESTOR_PRESENT
+#define SL_CATALOG_MATTER_POWER_SOURCE_PRESENT
+#define SL_CATALOG_MATTER_SOFTWARE_DIAGNOSTICS_PRESENT
+#define SL_CATALOG_MATTER_TIME_FORMAT_LOCALIZATION_PRESENT
+#define SL_CATALOG_MATTER_USER_LABEL_PRESENT
+#define SL_CATALOG_MATTER_BLE_PRESENT
+#define SL_CATALOG_MATTER_WIFI_NETWORK_DIAGNOSTICS_PRESENT

```

Comment:
You need the matter component tied to Studio zap cluster. I don't think it breaks anything but it could be cleaner without

Reply:
sure, will remove these.